src/requests/getImages.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+const getImages = (query) => {
+  if (!query) {
+    return Promise.resolve([]);
+    // eslint-disable-next-line no-else-return
```

Comment:
See comment in `SearchResults` regarding escaping rules that dont need to be escaped